### ISSUE TITLE
mib2dev prints enum text instead of numerical values

### ISSUE THREAD

**Issue by cristianpsg** [etingof/snmpsim#27]
Hi,

I'm trying to use mib2dev to generate some simulation data, however when I run it is printing the enum text values instead of the numerical ones.
The problem with this is that when I try to use this file on the simulator it complains because the data doesn't match the type (is expecting numbers and not text I presume).

There is any way to tell the script to print the integer values instead of the text?

Thanks.

1.3.6.1.4.1.13712.791.21.1.1.1.1.1.0|66|23522
1.3.6.1.4.1.13712.791.21.1.1.1.2.1.1.0|2|**waitingForAcquisition**
1.3.6.1.4.1.13712.791.21.1.1.1.2.1.2.0|2|**txInhibited**
1.3.6.1.4.1.13712.791.21.1.1.1.2.2.1.0|4|quaintly
1.3.6.1.4.1.13712.791.21.1.1.1.2.2.2.0|2|**geographicRestrictions**
1.3.6.1.4.1.13712.791.21.1.1.1.2.3.1.0|64x|3cb42955
1.3.6.1.4.1.13712.791.21.1.1.1.2.3.2.0|66|27459
1.3.6.1.4.1.13712.791.21.1.1.1.3.1.1.0|2|**true**
1.3.6.1.4.1.13712.791.21.1.1.1.3.2.1.0|2|**warning**

**Comment by etingof** [comment]
Thank you for reporting this issue!

Could you please try current master  (commit  cd236ce) and let me know if it works for you?

**Comment by cristianpsg** [comment]
Hi,

Thank you for looking at this so quickly, by the way, great work!

I tested it and it doesn't print the enums text anymore, however, the way is being done, it will produce values that are out of the ranges defined on the enums.
As an example, I for the OID below I got a value higher than 2 and wasn't happening before your update.

I'm not I python programmer but something like the code below seems to fix this issue, so pretty much is creating the value based on the syntax provided and then is just extracting the numerical representation.
I'm sure there must be a cleaner way to do this, but at least for now this seems to work with the MIB file I'm currently using.

Again thank you for your help and keep up with the good work.

```
            if syntax.tagSet == rfc1902.Integer32.tagSet:
                return rfc1902.Integer32(syntax.clone(val))

            if syntax.tagSet == rfc1902.Unsigned32.tagSet:
                return rfc1902.Unsigned32(syntax.clone(val))

            if syntax.tagSet == rfc1902.Bits.tagSet:
                return rfc1902.OctetString(syntax.clone(val))
```

```
honAkTxCtrlState  OBJECT-TYPE
    SYNTAX      HonAkDiscreteInputState
    MAX-ACCESS  read-only
    STATUS      current
    DESCRIPTION "Indicates whether or not satellite transmission control
                 analog input to KANDU is enabled.

                 ground(1): Transmission is disabled.
                 open(2):   Transmission is enabled."
    ::= { honAkInService  12 }
```

**Comment by etingof** [comment]
You are right, that's my bug. Fixed in 0a8e84be7319623126cb43a7e963d099bc42023f,

Thank you! ;-)
